src/vmm/src/mmds/data_store.rs

```diff
@@ -65,19 +65,20 @@ pub enum MmdsDatastoreError {
 // Used for ease of use in tests.
 impl Default for Mmds {
     fn default() -> Self {
-        Self::default_with_limit(51200)
+        Self::try_new(51200).unwrap()
     }
 }
 
 impl Mmds {
     /// MMDS default instance with limit `data_store_limit`
-    pub fn default_with_limit(data_store_limit: usize) -> Self {
-        Mmds {
+    pub fn try_new(data_store_limit: usize) -> Result<Self, MmdsDatastoreError> {
+        Ok(Mmds {
+            version: MmdsVersion::V1,
             data_store: Value::default(),
-            token_authority: None,
+            token_authority: TokenAuthority::try_new()?,
             is_initialized: false,
             data_store_limit,
-        }
+        })
```

Comment:
nit: also unrelated to this PR, but 
```suggestion
        Ok(Mmds {
            token_authority: TokenAuthority::try_new()?,
            data_store_limit,
            .. Default::default()
        })
```
would mean we only have a single source of truth regarding the "default mmds version" (the Default impl for MmdsVersion).

Reply:
ah oops, didn't realize `Mmds::default()` just calls `try_new()` :see_no_evil: Using `MmdsVersion::default()` addresses the spirit of my complaint :D